composer.json

```
@@ -9,13 +9,13 @@
   "prefer-stable": true,
   "minimum-stability": "dev",
   "require": {
-    "php": "^8.1",
-    "pimcore/pimcore": "^11.0",
+    "php": "^8.1.0 || ^8.2.0 || 8.3.0 || ^8.4.0",
```

```suggestion
    "php": "~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0",
```

`^` is wrong, because this allow also 8.5, 8.6, ...